server/refishing-fst2.txt

```diff
@@ -0,0 +1,30 @@
+#!/usr/local/bin/foma -f
+# Basic FST for the Germanic dataset
+
+# German
+define German [] ;
+
+clear
+push German
+save stack german.bin
+
+# Dutch
+define Dutch [] ;
+
+clear
+push Dutch
+save stack dutch.bin
+
+# English
+define English [] ;
+
+clear
+push English
+save stack english.bin
+
+# Proto-Germanic
```

Comment:
You can remove this and the other mentions once it's gone from the input data.

Reply:
Also the refishing code is a real mess... it is mostly untouched from the original form I received it in. I'm not sure if you want to take a look at it and try and clean it up, realistically I think it is deserving of an entire rewrite, which is beyond the scope of my understanding. I don't like that it was using a separate `lexicon.tsv` file, when I believe the refishing should always just rely on the existing data from the boards, so you will see in `refish.py` that I've commented this out.

Reply:
Okay, I will try to look into this!